app/src/main/res/layout/session_list_item.xml

```diff
@@ -76,7 +76,7 @@
         android:adjustViewBounds="true"
         android:baseline="13sp"
         android:contentDescription="@string/session_list_item_without_video_content_description"
-        android:src="@drawable/ic_without_video_recording"
+        android:src="@drawable/ic_novideo"
```

Comment:
# Schedule screen: camera icon
I only now noticed a **little vertical offset** between the title text and the camera icon. You can see it pretty well when you enable layout bounds in the developer settings.

![20230120_schedule_screen_camera_icon_phone](https://user-images.githubusercontent.com/144518/213742877-f72d5928-a02b-4b14-8f2e-6ac50d71383a.png)

Can we please fix this?

Reply:
This should be fixed now, although I couldn't find the option you mentioned

Reply:
Go into "Settings" -> "System" -> "Developer Options" (must be activated before) and there search for "Show layout bounds".

Reply:
Thanks, I actually forgot to update this for Tablets, will be fully fixed in next push

Reply:
Can we position the icons in the **top right** corner matching the **first row** of the single or multiline text? 

I also noticed that the camera and bell icons appear **smaller** for a single line title and **bigger** for a multiline title. Can we always use the same size? This can be seen in the 36C3 app in the "Chaos-West Bühne" column on a phone in landscape mode.

![icons1](https://user-images.githubusercontent.com/144518/214436352-45b51c85-4542-4ef1-88b9-6e0772b764f7.png) ![icons2](https://user-images.githubusercontent.com/144518/214436356-6a9e6d5d-c074-404c-8ba4-d9e10d569368.png)

Reply:
Always using the same size is no Problem, using the Top Right also isn't a problem, the only weird thing is the size. I think I've got the correct size now, but somehow if I use the defined Text Size (16sp) as the height for the Icons, they seem a lot smaller. I am currently using about 1.375x the text size which seems to be the correct height, but I find this very strange. It's probably some padding added to the text, but I don't see where that would be defined :(

Reply:
In commit 48124f565735f188980bf5243877de1119ed29ee the size difference is still present in landscape mode. For you, too?

Reply:
Which Device / Screen?

Reply:
Fixed

Reply:
I used a Pixel 6, Android 13 in landscape mode.

Reply:
Nice, I also tested on that, should be fixed now :+1:

Reply:
Looks good on my Pixel 6. Can you please document why the `layout_height` values are needed resp. how you found them? Also I would be interested what `fitEnd` vs `fitStart` changes for the icons.

Reply:
I can add comments that the Values values were found by trial and error and that they generally are 1,725x the old value for the old grid-based Values. Probably something like this: 

```
<!-- Scale adjusted by 1.725x to match font size while switching to Vector Graphics (Value found by Trial and Error) -->
```

`fitStart` makes the Icons render at the start of their Bounding box if they don't completely fill it (in this case the relevant thing is that they render at the top), while `fitEnd` makes them render at the End of said box. I hope you didn't find any instances of me using `fitEnd`? If so, that probably wasn't intentional

Reply:
The commit message details is a good place for the explanation.

Reply:
Added :+1: